metadata/transportation/road_centerlines

```diff
@@ -0,0 +1,84 @@
+# Title
+
+Utah Roads
+
+## ID
+
+a6cf0b81-a99e-4ec3-a9b7-7372ae6a6cbb
+
+## Brief Summary
+
+Statewide road centerline data for Utah.
```

Comment:
**nitpick**: I don't love when the word is used in the definition. Centerline is a non standard word. 

The first sentence of the summary is a significant improvement. 

> Utah Roads is a multi-purpose statewide roads dataset for cartography, routing, and range-based address location.

Reply:
I can understand that with definitions. I've been doing some reading on how the Bureau of Transportation Statistics explains their road data, I'll be making some adjustments to the language from that and we can see what we think.